docs/api/IoT.js-API-File-System.md

```diff
@@ -112,6 +112,161 @@ fs.closeSync(fd);
 ```
 
 
+## Class: fs.ReadStream
```

Comment:
The new elements are not documented at the start of the file where the API table is.